src/components/shell-panel/shell-panel.scss

```diff
@@ -20,8 +20,23 @@
     flex
     flex-initial
     items-stretch;
-  --calcite-shell-panel-detached-max-height: unset;
   z-index: var(--calcite-shell-panel-z-index, theme("zIndex.default"));
+  --calcite-shell-panel-detached-max-height: unset;
```

Comment:
@macandcheese should we rename these CSS props to not use detached and float instead?

Reply:
Yeah we should update to the final value name. At some point we can probably look to combine this with the other css vars, maybe when we look at the content-behind / Shell cases.